### PR TITLE
dm: print the dashboard address if OpenAPI enabled

### LINKED ISSUE #6643: Show DM dashboard URL while master server starting
### Is your feature request related to a problem?

Users don't know the dashboard URL even though we enabled the OpenAPI feature.

```go
	if s.cfg.OpenAPI {
		// tls3 is used to openapi reverse proxy
		tls3, err1 := toolutils.NewTLS(s.cfg.SSLCA, s.cfg.SSLCert, s.cfg.SSLKey, s.cfg.AdvertiseAddr, s.cfg.CertAllowedCN)
		if err1 != nil {
			return terror.ErrMasterTLSConfigNotValid.Delegate(err1)
		}
		if initOpenAPIErr := s.InitOpenAPIHandles(tls3.TLSConfig()); initOpenAPIErr != nil {
			return terror.ErrOpenAPICommonError.Delegate(initOpenAPIErr)
		}
		userHandles["/api/v1/"] = s.openapiHandles
		userHandles["/dashboard/"] = ui.InitWebUIRouter()
	}
```

### Describe the feature you'd like

It's better to print the dashboard URL to console.the 

### Describe alternatives you've considered

_No response_

### Teachability, Documentation, Adoption, Migration Strategy

_No response_

### PR DESCRIPTION
**Pull request by lonng**
### What problem does this PR solve?

Issue Number: close #6643

### What is changed and how it works?

Add a log to print the dashboard address.

#### Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)

```
 tiflow git:(issue6643) ✗ bin/dm-master --openapi --master-addr 127.0.0.1:8261 | grep Web
[2022/08/09 11:44:15.270 +08:00] [INFO] [server.go:222] ["Web UI enabled"] [dashboard=http://127.0.0.1:8261/dashboard/]
```

### Release note 

```release-note
print the dashboard address in log if OpenAPI enabled
```

### CODE REVIEW

**Review by lance6716 — APPROVED** [review]
/cc @Ehco1996

**Review by lichunzhu — APPROVED** [review]
LGTM

File: dm/master/server.go [hunk 1]
```diff
@@ -213,8 +213,17 @@ func (s *Server) Start(ctx context.Context) (err error) {
 		if initOpenAPIErr := s.InitOpenAPIHandles(tls3.TLSConfig()); initOpenAPIErr != nil {
 			return terror.ErrOpenAPICommonError.Delegate(initOpenAPIErr)
 		}
+
+		const dashboardPrefix = "/dashboard/"
+		scheme := "http://"
+		if len(s.cfg.SSLCA) != 0 {
```
Comment (Ehco1996) [inline dm/master/server.go:219]:
```suggestion
		if tls3.TLSConfig()!=nil {
```
see comment in L208

**Review by Ehco1996 — COMMENTED** [review]
restLGTM

**Review by Ehco1996 — APPROVED** [review]

**Pull request merged**